Samplero/Samplero/Screen/ShopBasket/View/Cell/ShopBasketCollectionViewCell.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  ShopBasketCollectionViewCell.swift
+//  Samplero
+//
+//  Created by DaeSeong on 2022/10/12.
+//
+
+import UIKit
+
+
+private enum Size {
+    static let sampleImageSize = 80.0
```

Comment:
p5.
`UIImage`를 작명했던것 처럼 굳이 `size`라는 것을 뒤에 명시하지 않아도 좋을 거같아요

```suggestion
    static let sampleImage = 80.0
```

Reply:
반영했습니다!